librz/util/print.c

```diff
@@ -512,56 +512,180 @@ RZ_API void rz_print_section(RzPrint *p, ut64 at) {
 	}
 }
 
-RZ_API void rz_print_hexdump(RzPrint *p, ut64 addr, const ut8 *buf, int len, int base, int step, size_t zoomsz) {
-	rz_return_if_fail(p && buf && len > 0);
-	PrintfCallback printfmt = (PrintfCallback)printf;
-#define print(x) printfmt("%s", x)
-	bool c = p ? (p->flags & RZ_PRINT_FLAGS_COLOR) : false;
-	const char *color_title = c ? (Pal(p, offset)
-					      : Color_MAGENTA)
-				    : "";
-	int inc = p ? p->cols : 16;
-	size_t i, j, k;
+static inline void print_addr(RzStrBuf *sb, RzPrint *p, ut64 addr) {
```

Comment:
please do the same suggested before for rz_print_section but for rz_print_addr and for possibly other code you have duplicated.